community/lexicon/schema/getSchemas.json

```diff
@@ -0,0 +1,48 @@
+{
+  "lexicon": 1,
+  "id": "community.lexicon.schema.getSchemas",
+  "defs": {
+    "main": {
+      "type": "procedure",
+      "description": "Gets schema views for a specified list of lexicon schemas (by AT-URI, with optional CID).",
+      "input": {
+        "encoding": "application/json",
+        "schema": {
+          "type": "object",
+          "required": ["refs"],
+          "properties": {
+            "refs": {
+              "type": "array",
+              "description": "List of schema refs (AT-URI with optional CID) to return hydrated views for.",
```

Comment:
It would be nice to encode this type of "at uri with optional cid" as a shared type as I think it's probably valuable in many use cases. Naming is hard tho, `community.lexicon.meta.optionallyStrongRef`? lol

Reply:
`weakRef`?

Reply:
It might be strong tho if the CID is included!